Assets/SEE/Game/City/NodeScalingAnimator.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace SEE.Game.City
+{
+    public class NodeScalingAnimator : MonoBehaviour
```

Comment:
What's the purpose of a `NodeScalingAnimator`? All animation must be handled by `NodeOperator`.